e2e/images_test.go

```diff
@@ -72,8 +76,77 @@ func TestImageRm(t *testing.T) {
 		icmd.RunCmd(cmd).Assert(t, icmd.Success)
 
 		expectedOutput := "REPOSITORY TAG APP NAME\n"
-		cmd.Command = dockerCli.Command("app", "image", "ls")
-		result := icmd.RunCmd(cmd).Assert(t, icmd.Success)
-		assert.Equal(t, result.Stdout(), expectedOutput)
+		expectImageListOutput(t, cmd, expectedOutput)
+	})
+}
+
+func TestImageTag(t *testing.T) {
+	runWithDindSwarmAndRegistry(t, func(info dindSwarmAndRegistryInfo) {
+		cmd := info.configuredCmd
+		dir := fs.NewDir(t, "")
+		defer dir.Remove()
+
+		// given a first available image
+		cmd.Command = dockerCli.Command("app", "bundle", filepath.Join("testdata", "simple", "simple.dockerapp"), "--tag", "a-simple-app", "--output", dir.Join("simple-bundle.json"))
+		icmd.RunCmd(cmd).Assert(t, icmd.Success)
+
+		singleImageExpectation := `REPOSITORY   TAG    APP NAME
+a-simple-app latest simple
+`
+		expectImageListOutput(t, cmd, singleImageExpectation)
```

Comment:
Is this assertion really useful? It's tested in the list test.

Reply:
Is this mandatory, I don't think so.
But I find it useful because when you read the test it shows the initial state and helps to understand what the test is doing.

Reply:
👍